### DISCUSSION TITLE
42Dash 로고 나왔습니당

### DISCUSSION THREAD

**Discussion by Likilee** [innovationacademy-kr/42dashboard#302] [General]
![dashboard42_logo_final](https://user-images.githubusercontent.com/35288028/180767821-b5356c7a-1148-431b-8186-275ad9346d09.png)

**Comment by Likilee** [comment]
![dashboard42_logo_final](https://user-images.githubusercontent.com/35288028/180769527-59789ec3-c28d-4797-ae59-e4b5bb7d145e.svg)
svg 버전

**Reply by jungmyeong96** [reply]
넘나리 이쁜것ㅎㅎㅎ

**Reply by hunjin-choi** [reply]
너무 멋집니다 😄

**Comment by Likilee** [comment]
svgr 을 사용해서 svg 이미지 사용하기
https://velog.io/@juno7803/React-React%EC%97%90%EC%84%9C-SVG-%ED%99%9C%EC%9A%A9%ED%95%98%EA%B8%B0
